charts/sscs-tya-notif/values.aat.template.yaml

```diff
@@ -15,5 +15,10 @@ java:
     PDF_SERVICE_BASE_URL: ${PDF_SERVICE_BASE_URL}rs/render
     PDF_SERVICE_ACCESS_KEY: ${PDF_SERVICE_ACCESS_KEY}
     PDF_SERVICE_HEALTH_URL: ${PDF_SERVICE_BASE_URL}rs/status
+    JOB_SCHEDULER_DB_PORT: "5432"
+    JOB_SCHEDULER_DB_PASSWORD:  ${JOB_SCHEDULER_DB_PASSWORD}
+    JOB_SCHEDULER_DB_USERNAME: "notification@sscs-tya-notif-postgres-db-aat"
```

Comment:
these should be in values.yaml with `{{ .Values.global.environment }}` in place of AAT

Reply:
```suggestion
    JOB_SCHEDULER_DB_USERNAME: "notification@sscs-tya-notif-postgres-db-{{ .Values.global.environment }}"
```